context-propagation/gradle.lockfile

```diff
@@ -1,51 +0,0 @@
-# This is a Gradle generated file for dependency locking.
```

Comment:
`1.0.x` should have this lockfile (`main` does not). Could you please revert deleting it?
You should be able to build without lockfile modifications since you don't have any dependency change.

Reply:
I guess the rebase + change base somehow didn't bring back the removed file. The lockfile is back.